spec/authentication.yml

```diff
@@ -726,6 +728,49 @@ components:
             description: "Error connecting conjur to the OIDC provider"
 
 
+    JWTAuthenticate:
+      parameters:
+        - $ref: 'openapi.yml#/components/parameters/RequestID'
+      post:
+        tags:
+          - "authentication"
+        summary: |
+          Gets a short-lived access token for applications using JSON Web Token (JWT)
+          to access the Conjur API.
+        description: |
+          Use the JWT Authenticator to leverage the identity layer
+          provided by JWT to authenticate with Conjur.
+        operationId: "getAccessTokenViaJWT"
+        parameters:
+          - name: "account"
+            in: "path"
+            required: true
+            description: "Organization account name"
+            schema:
+              $ref: 'openapi.yml#/components/schemas/AccountName'
+
+          - $ref: openapi.yml#/components/parameters/ServiceID
+
+        requestBody:
+          description: "ID token"
+          required: true
+          content:
+            application/x-www-form-urlencoded:
+              schema:
+                $ref: '#/components/schemas/JWTToken'
```

Comment:
Looks like this schema is not defined anywhere, you must define this object type in the `schemas` section of this file. Similar to how `AccessToken` is defined.